homeworks/src/ua/kstt/justymenko/day8maps/Solution.java

```diff
@@ -0,0 +1,30 @@
+package ua.kstt.justymenko.day8maps;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+/**
+ * Created by n1kropol on 4/21/17.
+ */
+public class Solution {
+    public static void main(String []argh){
+        Scanner in = new Scanner(System.in);
+        Map dicMap = new HashMap<String, Integer>();
```

Comment:
try not to use raw data types 
`Map<String, Integer> phoneBook = new HashMap<>();`  - since java 7